sdk/batch/arm-batch/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "BatchManagementClient Library with typescript type definitions for node.js and browser.",
   "version": "4.0.0",
```

Comment:
Version should be 5.0.0
You can either update this manually or pass the version when running the code generator too.

If you are doing the update manually, also update the version in batchManagementClientContext.ts file

Reply:
Fixed.